apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -62,10 +63,19 @@ class ReviewTab extends Component {
     navigateToHref(this.generateLevelUrlWithCodeReviewParam());
   };
 
-  generateLevelUrlWithCodeReviewParam = () =>
-    currentLocation().origin +
-    currentLocation().pathname +
-    `?${VIEWING_CODE_REVIEW_URL_PARAM}=true`;
+  generateLevelUrlWithCodeReviewParam = () => {
+    let url =
+      currentLocation().origin +
+      currentLocation().pathname +
+      `?${VIEWING_CODE_REVIEW_URL_PARAM}=true`;
+
+    // If teacher account is viewing as student, set up URLs
+    // to persist this setting when they click to view another project.
+    if (this.props.userIsTeacher && !this.props.viewAsTeacher) {
+      url += `&viewAs=Participant`;
```

Comment:
To clarify - the `viewAs=Participant` flag is not necessary for a participant viewing another participant's project? i.e. student viewing another student's in their section?

Reply:
Yeah exactly, it's only relevant if you are logged in with a teacher account and see the blue pullout on the right, which changes the URL (and your view of code review, among other things). Here's a teacher account looking at their own work on a Javalab level with Student selected (looks good, looks like expected experience for a participant/student):

![image](https://user-images.githubusercontent.com/25372625/163607915-598f0cea-59be-408a-9285-40c4e79c89ed.png)

Here's teacher account looking at a Javalab level with View as Teacher selected (looks weird, assumes that you are viewing a student's work -- eg, puts your name where it expects a students name in "This is the beginning of feedback for benjamin's project):

![image](https://user-images.githubusercontent.com/25372625/163608116-64fe11d1-0f0d-49d3-9407-2eb64213f8ec.png)